### PR TITLE
test: add docs, add Nox config, run in CI

### PR DESCRIPTION
**Pull request by scop**
See individual commits for details.

### CODE REVIEW

**Comment by scop** [comment]
Note: tests will fail on < 3.9 until #16 is addressed.

**Review by guludo — COMMENTED** [review]
I'm okay with merging, but, as commented below, I wonder if we really need to support python<3.6.

File: noxfile.py [hunk 1]
```diff
@@ -0,0 +1,14 @@
+"""nox config for venv-run."""
+
+import nox
+
+
+@nox.session(
+    python=["{}3.{}".format(py, x) for py in ("", "pypy") for x in range(5, 13)]
```
[the remark this answers is not in the document]
Comment (scop) [reply noxfile.py:7]: Sure, 3.5 support is why I went with `.format` here (and 3.5 support coming from `setup.py`).

My feeling is that venv-run is mostly a development tool, and even if some project that uses it wishes to support older Python versions for their project, they are very likely to develop it with an up to date one as far as venv-run goes. I guess I'd drop support for all EoL versions and go with >=3.7, there's always the older versions available for setups wanting support for older.

[the remark this answers is not in the document]
Comment (scop) [reply noxfile.py]: #20 drops < 3.7 support, rebased this on top of it, and addressed the above.

**Review by guludo — APPROVED** [review]
LGTM

**Pull request merged**